civis/ml/tests/test_model.py

```diff
@@ -451,12 +471,12 @@ def test_table_None(mock_res, mock_lt):
 
 @mock.patch.object(_model.cio, "file_id_from_run_output",
                    mock.Mock(return_value=11, spec_set=True))
-@mock.patch.object(_model.cio, "file_to_json", return_value='bar')
+@mock.patch.object(_model.cio, "file_to_json", return_value={'foo': 'bar'})
 @mock.patch.object(_model.ModelFuture, "_set_model_exception")
 def test_metadata(mock_spec, mock_f2j):
     c = setup_client_mock(3, 7)
     mf = _model.ModelFuture(3, 7, client=c)
-    assert mf.metadata == 'bar'
+    assert mf.metadata == {'foo': 'bar'}
```

Comment:
Why make this change?  Is this because the test before the change raises a warning after these changes?

Reply:
The metadata property now assumes that the metadata is always in the form of a dictionary. It calls `get` on the object. This is in L523 of `_model.py`.